aries_cloudagent/config/argparse.py

```diff
@@ -761,11 +773,19 @@ def get_settings(self, args: Namespace) -> dict:
                 settings["ledger.genesis_file"] = args.genesis_file
             elif args.genesis_transactions:
                 settings["ledger.genesis_transactions"] = args.genesis_transactions
+            elif args.genesis_transactions_list:
+                with open(args.genesis_transactions_list, "r") as stream:
+                    txn_config_list = yaml.safe_load(stream)
+                    ledger_config_list = []
+                    for txn_config in txn_config_list:
+                        ledger_config_list.append(txn_config)
+                    settings["ledger.ledger_config_list"] = ledger_config_list
             else:
                 raise ArgsParseError(
-                    "One of --genesis-url --genesis-file or --genesis-transactions "
-                    "must be specified (unless --no-ledger is specified to "
-                    "explicitly configure aca-py to run with no ledger)."
+                    "One of --genesis-url --genesis-file, --genesis-transactions "
+                    "or --genesis-transactions-list must be specified (unless "
+                    "--no-ledger is specified to explicitly configure aca-py to"
+                    " run with no ledger)."
```

Comment:
One suggestion - allow the existing `--genesis-url`, `-genesis-file` etc and *in addition* allow the multi-ledger yaml file to be specified.  This supports any existing configurations but allows additional ledgers to be added.

If `--genesis-url`, `--genesis-file` etc is specified then this would be the `write_ledger`, otherwise would have to be specified in the multi-ledger yaml file